README.md

```diff
@@ -332,31 +339,29 @@ respective responsibilities of MVC.
 
 Don't worry about `assets`, `serializers`, `mailers`, or `helpers` for now.
 
-## Code Along: Let's do this
+### Demo: Building our first Rails App
```

Comment:
Corrected. Was from combining with single resource. Tried to make the sections between overview and single distinct but when moving back to separate repos, missed the correction.